doc/tags/macro.rst

```diff
@@ -75,7 +75,7 @@ via the ``from`` tag:
 .. caution::
 
     As macros imported via ``from`` are called like functions, be careful to
-    not override existing functions:
+    not shadow existing functions:
```

Comment:
I suggest saying `be careful that they shadow existing functions` (the current wording makes it look like it is an issue to perform such shadowing, but if you don't intent to use the function, there is no issue)

Reply:
Done